packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -266,5 +305,13 @@ const getStyles = (theme: GrafanaTheme2) => {
       color: theme.colors.text.secondary,
       opacity: 0.7,
     }),
+    description: css({
+      fontStyle: 'italic',
```

Comment:
I know it was in the other PR this came from, but we don't think the description should be italic. Otherwise, this looks good! 👍